File: server/php/src/Application/Domain/Payments/PaymentDetails/PaymentProduct3208SpecificOutput.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails;

final class PaymentProduct3208SpecificOutput
{
    public ?string $buyerCompliantBankMessage = null;

    public function toArray(): array
    {
        return [
            'buyerCompliantBankMessage' => $this->buyerCompliantBankMessage,
        ];
    }
}

File: server/ruby/app/business/domain/payments/payment_details/payment_product3208_specific_output.rb
module Business
  module Domain
    module Payments
      module PaymentDetails
        class PaymentProduct3208SpecificOutput
          attr_accessor :buyer_compliant_bank_message

          def initialize(buyer_compliant_bank_message: nil)
            @buyer_compliant_bank_message = buyer_compliant_bank_message
          end

          def to_h
            {
              'buyerCompliantBankMessage' => @buyer_compliant_bank_message
            }
          end
        end
      end
    end
  end
end